src/js/components/jtable/TableCell.jsx

```diff
@@ -24,34 +37,40 @@ export class TableCell extends Component {
             classNames.push(className);
         }
 
+        const {
+            linkProps,
+            tagOrComponent
+        } = generateLink('div', href, linkTo);
```

Comment:
It's not a random do everything container, it's a stand-in for `<td>` so that when something else comes up that's a problem in the future we can change how tables are generated without having to refactor every single component that uses tables.